.gitignore

```diff
@@ -11,3 +11,5 @@ dist/
 
 # cypress
 cypress/
+
+/.vscode/
```

Comment:
Please remove that, it's unrelated to the project. You should have a global gitignore on your system that ignores the `.vscode` folder everywhere.